### ISSUE TITLE
Trunk and extremity distribution to skin lesions

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9988]
**Preferred term label:**
Trunk and extremity distribution to skin lesions

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Wells syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
this seems a bundled term.
Possibly distal (https://hpo.jax.org/app/browse/term/HP:0012839) is synonymous or nearly so with "on the extremitiess). Axial includes the head, so maybe we need to create a term called Truncal to denote this spatial pattern?

**Comment by MickeySegal** [comment]
Agree.  I changed our finding to truncal distribution.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000425
- label: Truncal location
- parents: [Localized (HP:0012838)](https://hpo.jax.org/app/browse/term/HP:0012838)
- pmid: 
- def: Applies to an abnormality that is situated in (or predominant in) the trunk of the body. The trunk is the main part of a person's body, not including the head, legs, or arms.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9988
- synonyms:
